src/navigation/Link/Link.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styles from "./Link.module.css";
+import { Plus } from "../../icons";
+import classNames from "classnames/bind";
+const cx = classNames.bind(styles);
+
+interface LinkProps {
+  to: string;
+  id?: string;
+  target?: "_blank" | "_self" | "_parent" | "_top";
+  disabled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  children: React.ReactNode;
+}
+
+const Link: React.FC<LinkProps> = ({
```

Comment:
**nitpick:** Preferably use `PropsWithChildren<LinkProps>` instead of `React.FC`. React.FC is not a good choice these days: https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13